### PR TITLE
Add external script reference example to installation

### PR DESCRIPTION
**Pull request by seetdev**
Adding code examples for referencing the minimized idb file as an external script. This addresses  #184

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,23 +2,40 @@
...
-   1. [`unwrap`](#unwrap)
-   1. [`wrap`](#wrap)
-   1. [General enhancements](#general-enhancements)
-   1. [`IDBDatabase` enhancements](#idbdatabase-enhancements)
-   1. [`IDBTransaction` enhancements](#idbtransaction-enhancements)
-   1. [`IDBCursor` enhancements](#idbcursor-enhancements)
-   1. [Async iterators](#async-iterators)
-1. [Examples](#examples)
-1. [TypeScript](#typescript)
+- [IndexedDB with usability.](#indexeddb-with-usability)
```
Comment (jakearchibald) [inline README.md:5]: Why is all this changing?

Comment (seetdev) [reply README.md:5]: @jakearchibald Sorry about this.

I notice it was modified and did not notice it changed from `1.` to `-`. It must be the VS.code extensions, I thought it was just adding the additional headers for 

```
## Directly In a browser

### Using the modules method directly via unpkg:
```

I will revert those

Comment (seetdev) [reply README.md]: I reverted the changes to the TOC. Sorry for the beginner's mistake. I turn off the formatting extensions on this project on my workspace going forward

File: README.md [hunk 2]
```diff
@@ -45,6 +49,17 @@ Or, use it directly via unpkg:
 </script>
 ```
 
+### Using external script reference
+
+```html
+<script src="https://unpkg.com/idb@5.0.4/build/iife/index-min.js"></script>
```
Comment (jakearchibald) [inline README.md:55]:
```suggestion
<script src="https://unpkg.com/idb@5/build/iife/index-min.js"></script>
```

I'm changing the above to 5 so it doesn't need updated with every release. It mean there's a redirect in practice, but this isn't the main recommended way to use the library anyway.

**Review by jakearchibald — APPROVED** [review]
Thank you!

File: README.md (hunk 2 above)
Comment (jakearchibald) [reply README.md:55]: Oh, seems like I can't commit the change above. Can you accept it? Then I'll merge this.

Comment (seetdev) [reply README.md:55]: I resolved the conversation. Thanks

**Comment by jakearchibald** [comment]
Perfect

**Comment by ShawnMcClure** [comment]
Thanks - the external script reference example is very helpful, and completes the loop on examples.

**Pull request merged**